### PR TITLE
Add basic doc about settings

### PR DESCRIPTION
**Pull request by yohanboniface**

### CODE REVIEW

File: docs/docker.md [hunk 1]
```diff
@@ -31,3 +31,14 @@ volumes:
...
+### Create superuser
+
+With docker-compose, run `docker-compose run app /venv/bin/umap createsuperuser`
+
+
+### Custom settings
+
+Some basic settings are available through env vars (see https://github.com/umap-project/umap/blob/master/umap/settings/base.py),
+but if you need more custom ones (like custom OAuth configuration), the easiest
+way is to push a settings file to `/etc/umap/umap.conf`.
```
Comment (davidbgk) [inline docs/docker.md:44]:
```suggestion
way is to push a [settings file](settings.md) to `/etc/umap/umap.conf`.
```

File: docs/settings.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+Here are a few relevent settings for uMap.
+
+
+#### ALLOWED_HOSTS
+
+The hosts that uMap expects.
+`ALLOWED_HOSTS = ['umap.mydomain.org']`
+
+Can be set through env var too: `ALLOWED_HOSTS=umap.mydomain.org`
```
Comment (davidbgk) [inline docs/settings.md:16]: How can you pass a list through that env var?

File: docs/settings.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+Set it to the default language you want. `LANGUAGE_CODE = "it"`
+
+#### LEAFLET_LONGITUDE, LEAFLET_LATITUDE, LEAFLET_ZOOM
+
+Default longitude, latitude and zoom for the map
+
+#### MEDIA_ROOT
+
+Where uMap should datalayers and icons, must be consistent with your
```
Comment (davidbgk) [inline docs/settings.md:49]:
```suggestion
Where uMap should store your datalayers and icons, must be consistent with your
```

File: docs/settings.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+#### SECRET_KEY
+
+Must be defined to something unique and secret.
+
+#### SITE_URL
+
+The final URL of you instance, including the protocol:
+
+`SITE_URL="http://umap.org"`
```
Comment (davidbgk) [inline docs/settings.md:60]: This is not consistent with other examples to add double quotes for env vars.

File: docs/settings.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+
+
+#### SITE_NAME
+
+The name of the site, to be used in header and HTML title.
+
+
+#### STATIC_ROOT
+
+Where uMap should store static files (CSS, JS…), must be consistent with your
```
Comment (davidbgk) [inline docs/settings.md:77]:
```suggestion
Where uMap should find your static files (CSS, JS…), must be consistent with your
```

File: docs/settings.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+#### USER_AUTOCOMPLETE_FIELDS
+
+Which fields to search when autocompleting users (in permissions).
+Eg.: `USER_AUTOCOMPLETE_FIELDS = ["^username", "email"]`
+
+
+#### USER_DISPLAY_NAME
+
+Advanced setting for controling which user fields will be used for displaying
+they name on the application, depending on which fields you collect with your
```
Comment (davidbgk) [inline docs/settings.md:93]:
```suggestion
their name on the application, depending on which fields you collect with your
```

File: docs/settings.md [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+
+#### USER_URL_FIELD
+
+Which field to be used in URL for user. Must be a unique field.
+
+Eg.: `USER_URL_FIELD = "pk"`
+
+#### UMAP_ALLOW_ANONYMOUS
+
+Should uMap allows user without an account to create maps.
```
Comment (davidbgk) [inline docs/settings.md:105]: What is the default?

File: docs/settings.md [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+
+#### UMAP_SEARCH_CONFIGURATION
+
+Use it if you take control over the search configuration.
+
+See [search](install.md#search) for details.
+
+#### UMAP_READONLY
+
+Is your instance readonly ? Useful for server maintenance.
```
Comment (davidbgk) [inline docs/settings.md:154]:
```suggestion
Is your instance readonly? Useful for server maintenance.
```

**Review by davidbgk — APPROVED** [review]

File: docs/settings.md (hunk 5 above)
Comment (yohanboniface) [reply docs/settings.md:77]: Not sure about this change: when running `collectstatic`, Django will cp static to that STATIC_ROOT directory. And Django/uMap itself should not server static files, as you know for sure :)

Comment (davidbgk) [reply docs/settings.md:77]: You're right, for these standard settings I would redirect to the Django documentation.

**Pull request merged**